Mapify/Mapify.csproj

```diff
@@ -2,7 +2,7 @@
     <Import Project="../UserProperties.props" Condition="Exists('../UserProperties.props')" />
     <PropertyGroup>
         <TargetFramework>net48</TargetFramework>
-        <LangVersion>7.3</LangVersion>
+        <LangVersion>9</LangVersion>
```

Comment:
What was this bump necessary for?

Reply:
1>Locale.cs(30,105): Error CS8370 : Feature 'target-typed object creation' is not available in C# 7.3. Please use language version 9.0 or greater.
1>Locale.cs(63,39): Error CS8370 : Feature 'target-typed object creation' is not available in C# 7.3. Please use language version 9.0 or greater.

Reply:
`private static ReadOnlyDictionary<string, Dictionary<string, string>> mapSpecificTranslations = new (new Dictionary<string, Dictionary<string, string>>());`

Reply:
In that case, simply don't use the new style of constructor. We should take a C# version bump more seriously and do a proper upgrade that takes advantage of it, but not now.

Reply:
updated